unittest/factory/cost.cpp

```diff
@@ -28,7 +28,7 @@ namespace unittest {
 const std::vector<CostModelTypes::Type> CostModelTypes::all(CostModelTypes::init_all());
 const std::vector<CostModelNoFFTypes::Type> CostModelNoFFTypes::all(CostModelNoFFTypes::init_all());
 
-std::ostream& operator<<(std::ostream& os, CostModelTypes::Type type) {
+std::ostream &operator<<(std::ostream &os, CostModelTypes::Type type) {
```

Comment:
Also, revert the changes here please

Reply:
I had fixed this. @edantec please do not worry about it.

Reply:
done